apps/condo/domains/miniapp/components/AppIndex/B2BAppPage.tsx

```diff
@@ -64,6 +68,30 @@ export const B2BAppPage: React.FC<B2BAppPageProps> = ({ id }) => {
         router,
     ])
 
+    const shouldSendAnalytics =
+        !(contextLoading || contextError || appRoleLoading || appRoleError || isSupport || isAdmin)
+        && Boolean(appUrl && appRole && context)
+
+    useEffect(() => {
+        if (!shouldSendAnalytics) return
+
+        const visitAt = dayjs().toISOString()
```

Comment:
seems like `ed` is missing? `createdAt`, `deletedAt`, `visitedAt`?

Reply:
fixed